### PR TITLE
fix(*): fixable eslint-plugin-vuejs-accessibility issues [KHCP-11024]

### PR DESCRIPTION
**Pull request by portikM**
# Summary

Addresses: https://konghq.atlassian.net/browse/KHCP-11024

### CODE REVIEW

File: src/components/KSelect/KSelectItem.vue [hunk 1]
```diff
@@ -14,6 +14,7 @@
         :disabled="item.disabled === true ? true : undefined"
         type="button"
         :value="item.value"
+        @click="handleClick"
```
Comment (adamdehaven) [inline src/components/KSelect/KSelectItem.vue:17]: question: how was this working previously if this is required?

Comment (portikM) [reply src/components/KSelect/KSelectItem.vue:17]: It was previously bound to div on line 6. I moved it to button

File: src/components/KMultiselect/KMultiselectItem.vue [hunk 2]
```diff
@@ -6,6 +6,7 @@
     role="listitem"
   >
     <div
+      :aria-selected="item.selected"
```
Comment (adamdehaven) [inline src/components/KMultiselect/KMultiselectItem.vue:9]: question: should this go on the parent?

File: src/components/KSelect/KSelectItem.vue [hunk 3]
```diff
@@ -3,9 +3,9 @@
     :key="item.key"
     class="select-item"
     :data-testid="`select-item-${item.value}`"
-    @click="handleClick"
   >
     <div
+      :aria-selected="item.selected"
```
Comment (adamdehaven) [inline src/components/KSelect/KSelectItem.vue:8]: question: should this go on the parent?

File: src/components/KMultiselect/KMultiselectItem.vue (hunk 2 above)
Comment (portikM) [reply src/components/KMultiselect/KMultiselectItem.vue:9]: No, it belongs on the `role="option"` element. Same in KSelectItem

**Review by adamdehaven — APPROVED** [review]

**Pull request merged**